tools/unsupervised_dataset/Crawler/CQNode.py

```diff
@@ -0,0 +1,32 @@
+class CQNode:
```

Comment:
In general, it's good to have tests covering the actual logic. So maybe you can consider adding some corresponding tests for each source files.

Reply:
+1 Please add tests0 
Assume pylint has been run on this?
Also do we already have some format checker integrated, it is easier to have people's code more consistent.